www/less/directory.less

```diff
@@ -178,6 +178,13 @@
   z-index: 5;
 }
 
+.pfResult .profile_pic .photo{
+  background-image:    url('/avatar/ebatie2?s=large');
```

Comment:
let's try to get a generic image rather than one tied to a person.  and also avoid an absolute file path.

Save the image to `www/images` and reference it with `../images/default-profile.png`